lib/vanagon/platform/rpm.rb

```diff
@@ -45,15 +45,15 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}.#{project.noarch ? 'noarch' : @architecture}.rpm"
       end
 
-      def output_dir(target_repo = "products")
-        super
-      end
```

Comment:
If I recall correctly, there are already pe projects that either have `target_repo` set to `pc1` or empty. So we already have a variety of path layouts to deal with here. How do we handle that currently? I can investigate.